preview-src/pullRequestOverviewRenderer.ts

```diff
@@ -591,11 +591,24 @@ class ReviewNode {
 						});
 					}
 
+					let outdated = comments[0].position !== comments[0].original_position;
+
 					const diffView: HTMLDivElement = document.createElement('div');
 					diffView.className = 'diff';
 					const diffHeader: HTMLDivElement = document.createElement('div');
 					diffHeader.className = 'diffHeader';
-					diffHeader.textContent = comments[0].path;
+					const diffPath: HTMLSpanElement = document.createElement('span');
+					diffPath.className =  outdated ? 'diffPath outdated' : 'diffPath';
+					diffPath.textContent = comments[0].path;
+					diffPath.addEventListener('click', () => this.openDiff(comments[0]));
```

Comment:
should the event listener only be added if the comment isn't outdated?

Reply:
done